pkg/policy/policy.go

```diff
@@ -0,0 +1,44 @@
+package policy
+
+import (
+	"github.com/datreeio/datree/pkg/fileReader"
+	"gopkg.in/yaml.v3"
+)
+
+const defaultRulesYamlPath = "./pkg/policy/defaultRules.yaml"
+
+type DefaultRulesDefinitions struct {
+	ApiVersion string                  `yaml:"apiVersion"`
+	Rules      []DefaultRuleDefinition `yaml:"rules"`
+}
+
+type DefaultRuleDefinition struct {
+	ID               int                    `yaml:"id"`
+	Name             string                 `yaml:"name"`
+	UniqueName       string                 `yaml:"uniqueName"`
+	EnabledByDefault bool                   `yaml:"enabledByDefault"`
+	DocumentationUrl string                 `yaml:"documentationUrl"`
+	MessageOnFailue  string                 `yaml:"messageOnFailue"`
```

Comment:
@dimabru `MessageOnFailue` is containing the typo :)

Reply:
thanks :)